src/components/PostCard/PostHeader/PostHeader.js

```diff
@@ -9,24 +9,25 @@ import Highlight from 'components/Highlight'
 import FlagContent from 'components/FlagContent'
 import Icon from 'components/Icon'
 import { personUrl, topicUrl } from 'util/navigation'
+import { formatDatePair } from 'util/index'
 import { humanDate } from 'hylo-utils/text'
 import './PostHeader.scss'
 import { filter, isFunction, isEmpty } from 'lodash'
 import cx from 'classnames'
 
-const formatStartDate = (startTime) => {
+const formatStartDate = (startTime, from) => {
   const current = moment()
   let start = ''
   if (moment(startTime).isAfter(current)) {
-    start = moment(startTime).format('MMM D YYYY')
+    start = from
```

Comment:
I use passed-in formatted start/end dates instead of format them here. formatDatePair was used in events so I imported it so it can be used in requests, offers and resources.